From 0cdd3b77d46db0794ded0db220b066b333cfbb33 Mon Sep 17 00:00:00 2001
From: Mohammed Sadiq <sadiq@sadiqpk.org>
Date: Wed, 2 Feb 2022 15:48:28 +0530
Subject: [PATCH] wifi: Wrap Hotspot info label

So that the window will fit small screens when the string
is localized

Fixes #14
---
 panels/network/cc-wifi-panel.ui | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panels/network/cc-wifi-panel.ui b/panels/network/cc-wifi-panel.ui
index 783012728..19455cf0d 100644
--- a/panels/network/cc-wifi-panel.ui
+++ b/panels/network/cc-wifi-panel.ui
@@ -244,6 +244,7 @@
                             <child>
                               <object class="GtkLabel">
                                 <property name="visible">True</property>
+                                <property name="wrap">True</property>
                                 <property name="label" translatable="yes">Mobile devices can scan the QR code to connect.</property>
                               </object>
                             </child>
-- 
2.34.1

